.gitpod.Dockerfile

```diff
@@ -16,7 +16,8 @@ RUN sudo apt-get update \
     sox gfortran \
  && sudo rm -rf /var/lib/apt/lists/*
 
-RUN cd ~/GitHub/r-with-intel-mkl/ \
+RUN mkdir ~/r-with-intel-mkl \
+ && cd ~/r-with-intel-mkl/ \
```

Comment:
what is the purpose of this directory? Don't see used it anywhere

Reply:
Yeah, I see it's not used. I will delete this.